packages/form/src/FormAutocomplete/FormAutocomplete.test.tsx

```diff
@@ -226,7 +226,7 @@ describe('FormAutocomplete', () => {
 
       renderWithTheme(<TestComponent />);
 
-      const setTextBtn = screen.getByRole('button');
+      const setTextBtn = screen.getByText('Set text');
```

Comment:
```suggestion
      const setTextBtn = screen.getByRole('button', { name: 'Set text' });
```
поробуй такой вариант